manifest.yaml

```diff
@@ -64,7 +64,7 @@ box:
     strategy:
       download: bucket
       project: mistserver
-      commit: 5c91f691ab11f2f5206b1ded49f945403cd70099
+      commit: cdd9408ffae8e0760a8d01d65b5df33bc0da7bb4
```

Comment:
https://github.com/livepeer/mistserver/commits/catalyst/
3 commits since `5c91f691`:
* [Use correct syntax for WebRTC Link headers](https://github.com/livepeer/mistserver/commit/fc74525bbb014d5bd83e5a60a850702cd375f7df)
* [Conditionally enable correct Link header syntax in WHIP/WHEP only when detecting OBS Studio or a modern Livepeer player](https://github.com/livepeer/mistserver/commit/ba7d6abf8feb1c5c7d09194a97bbc8322f93a396)
* [Persist Livepeer process cookies as much as possible, but reset them when attempting to switch broadcasters on purpose.](https://github.com/livepeer/mistserver/commit/cdd9408ffae8e0760a8d01d65b5df33bc0da7bb4)